src/components/molecules/definitionitem/__tests__/DefinitionItem.tests.js

```diff
@@ -0,0 +1,192 @@
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar MB02',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar HD204',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar BAT1706',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar HLX04',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar IBI305',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar SCT510',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar LY01008',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar CT-P16',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar RPH-001',
+		},
+		{
+			type: 'Synonym',
+			name: 'bevacizumab biosimilar TRS003',
+		},
+		{
+			type: 'CodeName',
+			name: 'HD204',
+		},
+		{
+			type: 'Synonym',
+			name:
+				'immunoglobulin G1 (human-mouse monoclonal rhuMab-VEGF gamma-chain anti-human vascular endothelial growth factor), disulfide with human-mouse monoclonal rhuMab-VEGF light chain, dimer',
+		},
+	],
+	definition: {
+		html:
+			'A recombinant humanized monoclonal antibody directed against the vascular endothelial growth factor (VEGF), a pro-angiogenic cytokine.  Bevacizumab binds to VEGF and inhibits VEGF receptor binding, thereby preventing the growth and maintenance of tumor blood vessels. Check for <a ref="https://www.cancer.gov/about-cancer/treatment/clinical-trials/intervention/C2039">active clinical trials</a> using this agent. (<a ref="https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI%20Thesaurus&code=C2039">NCI Thesaurus</a>)',
+		text:
+			'A recombinant humanized monoclonal antibody directed against the vascular endothelial growth factor (VEGF), a pro-angiogenic cytokine.  Bevacizumab binds to VEGF and inhibits VEGF receptor binding, thereby preventing the growth and maintenance of tumor blood vessels. Check for active clinical trials using this agent. (NCI Thesaurus)',
+	},
+	drugInfoSummaryLink: {
+		uri: 'https://www.cancer.gov/about-cancer/treatment/drugs/bevacizumab',
+		text: 'Bevacizumab',
+	},
+	nciConceptId: 'C2039',
+	nciConceptName: 'Bevacizumab',
+	termId: 43234,
+	name: 'bevacizumab',
+	firstLetter: 'b',
+	type: 'DrugTerm',
+	termNameType: 'PreferredName',
+	prettyUrlName: 'bevacizumab',
+};
+
+const dictionaryName = 'Cancer.gov';
+const dictionaryTitle = 'NCI Dictionary of Cancer Terms';
+useStateValue.mockReturnValue([
+	{
+		appId: 'mockAppId',
+		basePath: '/',
+		dictionaryName,
+		dictionaryTitle,
+		language: 'en',
+	},
+]);
+describe('Definition Item component', () => {
+	const wrapper = render(
+		<MockAnalyticsProvider>
+			<MemoryRouter initialEntries={['/bevacizumab']}>
+				<DefinitionItem payload={payload} />
+			</MemoryRouter>
+		</MockAnalyticsProvider>
+	);
+
+	test('Renders with child components [ Search | AZList ]', () => {
+		const { container } = wrapper;
+		expect(container.querySelector('.term-title')).toBeInTheDocument();
```

Comment:
Shouldn't this also check the contents? Otherwise there is no sense in using so much content for the mock...
(I think it should check contents)